Sources/Controllers/DashboardOnMap/MapSettings/OAMapSettingsWikipediaLanguagesScreen.mm

```diff
@@ -97,6 +95,7 @@ - (void)viewDidLoad
     self.tableView.dataSource = self;
     self.tableView.editing = YES;
     self.tableView.tintColor = UIColorFromRGB(color_primary_purple);
+    self.tableView.separatorInset = UIEdgeInsetsMake(0., [OAUtilities getLeftMargin] + 52., 0., 0.);
```

Comment:
This will probably be incorrect on device rotation. You need to call it again in ViewWillTransitionToSize